Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -663,8 +663,15 @@ class BrowserViewController: UIViewController {
         urlBar.currentURL = url
         urlBar.leaveOverlayMode()
 
-        if let tab = tabManager.selectedTab,
-           let nav = tab.loadRequest(NSURLRequest(URL: url)) {
+        guard let tab = tabManager.selectedTab else {
+            return
+        }
+
+        if #available(iOS 9.0, *) {
+            tab.webView?.customUserAgent = nil
```

Comment:
I think we want this to also include navigation from the page (clicking links, submitting forms, etc.), so I don't think `finishEditingAndSubmit` will be a wide enough net for what we need. How about moving this to `didCommitNavigation`?